src/styles/legends.js

```diff
@@ -0,0 +1,18 @@
+import legendAchromatic from './achromatic_legend.js';
+import legend3005 from './3005_legend.js';
+
+function selectLegend() {
+    const legendName = process.env.REACT_APP_LEGEND_FILE;
```

Comment:
This is very picky, but a point perhaps worth making anyway: 

The env vars are an interface, even in some sense a "user interface" and should accordingly focus on *what*, and *not expose the internal how*. 

So referring to a "file" is undesirable here because the user has no control over that file nor any business knowing it is a file -- which could easily change in future as we change our internal implementation. Instead we could call the env var something more what-focused like `REACT_APP_LEGEND_TYPE` and let it take on simpler values such as `3005` and `achromatic`. These are what would appear in, say, a drop-down, which is kind of the model for this kind of selection.